tests/test_effect_harmonization.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pyspark.sql.functions as f
+import pytest
+
+from etl.gwas_ingest.effect_harmonization import get_reverse_complement
+
+if TYPE_CHECKING:
+    from pyspark.sql import DataFrame, SparkSession
+
+
+@pytest.fixture
+def mock_allele_columns(spark: SparkSession) -> DataFrame:
+    return spark.createDataFrame(
+        [
+            {"allele": "A", "reverseComp": "T", "isPalindrom": False},
```

Comment:
There's a typo in the name of the column
```suggestion
            {"allele": "A", "reverseComp": "T", "isPalindrome": False},
```

Reply:
I don't know how it could be passed. Thank you for noticing.